src/radical/pilot/bootstrapper/bootstrap_1.sh

```diff
@@ -429,8 +429,8 @@ virtenv_setup()
     virtenv_mode="$3"
     python_dist="$4"
 
-    virtenv_create=UNDEFINED
-    virtenv_update=UNDEFINED
+    ve_create=UNDEFINED
+    ve_update=UNDEFINED
```

Comment:
This name change removes an ambiguity between function and variable names.  The shell parsing had no problem with that - human parsers did... ;)